faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -44,6 +44,8 @@ package query {
     // rather transform the value to a series of object constructions.
     implicit def quotedValue(unwrapped: Value) = Expr(ObjectV("quote" -> unwrapped))
     implicit def quotedResult(unwrapped: Result[Value]) = Expr(ObjectV("quote" -> unwrapped.get))
+
+    implicit def encode[T : Encoder](obj: T): Expr = Value.from(obj)
```

Comment:
Now that we have implicit encoders, it would be nice to remove the other ones. We could pattern match on the `Value` returned to determine the ones that need special escaping.